base_report_to_printer/models/ir_actions_report.py

```diff
@@ -161,3 +170,24 @@ def _render_qweb_pdf(self, report_ref, res_ids=None, data=None):
             )
 
         return document, doc_format
+
+    def _render_qweb_text(self, report_ref, docids, data=None):
+        """Generate a TEXT file and returns it.
+
+        If the action configured on the report is server, it prints the
+        generated document as well.
+        """
+        document, doc_format = super()._render_qweb_text(
+            report_ref, docids=docids, data=data
+        )
+
+        behaviour = self.behaviour()
```

Comment:
This is plain wrong in v16 ! self doesnt' contrain the report obj ; report_ref has it. FIxed in https://github.com/OCA/report-print-send/pull/316
When you make a PR, you should make sure that the code works.